packages/apollo-server-core/src/runHttpQuery.ts

```diff
@@ -298,6 +300,18 @@ export async function processHTTPRequest<TContext>(
         parseGraphQLRequest(httpRequest.request, requestParams),
       );
 
+      if (requests.length > 1 && options.allowBatchedHttpRequests === false) {
```

Comment:
I couldn't figure out a nice way of setting defaults in the GraphQLServer options so I've left this conditional naive for now - I'd be keen to improve this